packages/block-editor/src/components/writing-flow/use-clipboard-handler.js

```diff
@@ -72,22 +91,6 @@ export default function useClipboardHandler() {
 			const expandSelectionIsNeeded =
 				! shouldHandleWholeBlocks && ! isSelectionMergeable;
 			if ( event.type === 'copy' || event.type === 'cut' ) {
-				if ( ! hasMultiSelection() ) {
-					const { target } = event;
-					const { ownerDocument } = target;
-					// If copying, only consider actual text selection as selection.
-					// Otherwise, any focus on an input field is considered.
-					const hasSelection =
-						event.type === 'copy' || event.type === 'cut'
-							? documentHasUncollapsedSelection( ownerDocument )
-							: documentHasSelection( ownerDocument );
```

Comment:
I changed this line to `documentHasSelection( ownerDocument ) &&  ! ownerDocument.activeElement.isContentEditable` while also running it for the paste event as we did before #54543.